django_examples/cards_examples/views/main.py

```diff
@@ -38,9 +39,21 @@ def setup_cards(self):
                         'label': 'Sample', 'label_td': True},
                        {'value': 'Value 1-1'}),
                       {'field': '', 'title': 'this is hidden'})
-
         card.add_entry(value='Welcome using the table template2', label='Total', value_th=True)
 
+    def ajax_btn(self):
+        ajax_btn = HtmlMenu(request=self.request, template='button_group') \
+            .add_items(AjaxButtonMenuItem(button_name='message',
+                                          menu_display='',
+                                          font_awesome='fa fa-question',
+                                          css_classes='btn btn-sm btn-primary')).render()
+        return ajax_btn
+
+    def button_message(self, **_kwargs):
+        self.add_command('message', text='Buttons')
+        self.add_command('null')
+        return self.command_response()
```

Comment:
this just needs to be

return self.command_response('message', text='Buttons')